source/style-guide/markup/directives.txt

```diff
@@ -0,0 +1,77 @@
+==========
+Directives
+==========
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Directives provide a means to extend |rst| to format blocks of text
+without resorting to new syntax. They all start with two periods and a
+space and end with two colons.
+
+.. code-block:: rst
+
+   .. directive::
+
+.. seealso::
+
+   To learn more about Directives, read the entries in the:
+
+   - :rst:`docutils docs </restructuredtext.html#directives>`
+
+   - :sphinx-rst:`Sphinx docs </basics.html#directives>`
+
+Directives in Use
+~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Directive
+     - Description
+
+   * - :rst:`contents </directives.html#table-of-contents>`
+     - Local, current file only, table of contents
+
+   * - :rst:`container </directives.html#container>`
```

Comment:
[question]

Should we remove the info about `container` because it's not available in Snooty? (To my knowledge)